refnx/reduce/platypusnexus.py

```diff
@@ -722,8 +722,10 @@ def process(self, h5norm=None, lo_wavelength=2.5, hi_wavelength=19.,
 
         elif peak_pos is None:
             # use the auto finder, falling back to manual_beam_find
+            tol = kwds.get('peak_pos_tol', 0.01)
```

Comment:
peak_pos_tol needs to be added to the keywords in the function signature, and the docstring amended.